app/src/main/java/com/hfad/beeradviser/FindBeerActivity.java

```diff
@@ -28,9 +28,10 @@ public void onClickFindBeer(View view) {
         String beerType = String.valueOf(colorSpinnerView.getSelectedItem());
 
         //TODO: WRITE code to get recommendations from the BeerExpert class
+       String beerBrand = BeerExpert.getRecommendation(beerType);
```

Comment:
When you use the model, be sure to use the reference variable, rather than the class name, e.g., use `expert.getRecommendation()` instead of `BeerExpert.getRecommendation()`